src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -140,10 +141,10 @@ private function normalizeIdentifiers($id, ObjectManager $manager, string $resou
                 continue;
             }
 
-            $identifier = $identifiersMap[$propertyName] ?? $identifierValues[$i] ?? null;
+            $identifier = !isset($identifiersMap) ? $identifierValues[$i] ?? null : $identifiersMap[$propertyName] ?? null;
 
             if (null === $identifier) {
-                throw new InvalidArgumentException(sprintf('Invalid identifier "%s".', $id));
+                throw new PropertyNotFoundException(sprintf('Invalid identifier "%s", "%s" has not been found.', $id, $propertyName));
```

Comment:
I'm not sure about this but it seems more a `PropertyNotFound` then an `InvalidArgument`. Let me know if I have to revert this.